redfish-core/lib/hypervisor_system.hpp

```diff
@@ -167,77 +168,117 @@ inline bool extractHypervisorInterfaceData(
         {
             std::pair<boost::container::flat_set<IPv4AddressData>::iterator,
                       bool>
-                it = ipv4Config.insert(IPv4AddressData{});
+                v4Itr = ipv4Config.insert(IPv4AddressData{});
 
-            IPv4AddressData& ipv4Address = *it.first;
-            if (objpath.first == "/xyz/openbmc_project/network/hypervisor/" +
-                                     ethIfaceId + "/ipv4/addr0")
+            std::pair<boost::container::flat_set<IPv6AddressData>::iterator,
+                      bool>
+                v6Itr = ipv6Config.insert(IPv6AddressData{});
+
+            IPv4AddressData& ipv4Address = *v4Itr.first;
+            IPv6AddressData& ipv6Address = *v6Itr.first;
+
+            for (std::string protocol : {"ipv4", "ipv6"})
             {
-                idFound = true;
-                if (interface.first == "xyz.openbmc_project.Network.IP")
+                std::string ipAddrObj =
```

Comment:
Upstream bmcweb is doing this via:
https://gerrit.openbmc.org/c/openbmc/bmcweb/+/54267/2/redfish-core/lib/virtual_media.hpp

Reply:
I haven't seen this change in the downstream code anywhere. Probably this can be taken as a new commit!

Reply:
Since its based from 1020 branch , lets keep this as it is until we do a master refresh during 1040. Please note this comment and make changes at upstream commit